### PR TITLE
feat: pendle fix

### PR DESCRIPTION
**Pull request by jpsains**

### CODE REVIEW

File: packages/adapters-library/src/adapters/pendle/tests/snapshots/ethereum.prices.1.json [hunk 1]
```diff
@@ -85,5 +85,6 @@
         }
       }
     }
-  ]
+  ],
+  "rpcResponses": {}
```
Comment (jpsains) [inline packages/adapters-library/src/adapters/pendle/tests/snapshots/ethereum.prices.1.json:89]: @bergarces any ideas why this is empty?

Comment (jpsains) [reply packages/adapters-library/src/adapters/pendle/tests/snapshots/ethereum.prices.1.json:89]: ah i know why

Comment (jpsains) [reply packages/adapters-library/src/adapters/pendle/tests/snapshots/ethereum.prices.1.json:89]: no requests are made for this token price its one to one

File: packages/adapters-library/src/adapters/pendle/products/lp-token/pendleLpTokenAdapter.ts [hunk 2]
```diff
@@ -191,20 +182,17 @@ export class PendleLpTokenAdapter
...
-        blockTag: blockNumber,
-      })
-      .catch((e) => {
-        return oracle.getLpToSyRate(metadata.marketAddress, DURATION_30_MINS, {
-          blockTag: blockNumber,
-        })
-      })
+    const marketAddress = metadata.marketAddress
+
+    // missing block number atm
```
Comment (bergarces) [inline packages/adapters-library/src/adapters/pendle/products/lp-token/pendleLpTokenAdapter.ts:192]: Not sure what this comment means.

File: packages/adapters-library/src/adapters/pendle/products/principle-token/pendlePrincipleTokenAdapter.ts [hunk 3]
```diff
@@ -192,20 +183,17 @@ export class PendlePrincipleTokenAdapter
...
-        blockTag: blockNumber,
-      })
-      .catch((e) => {
-        return oracle.getYtToSyRate(metadata.marketAddress, DURATION_30_MINS, {
-          blockTag: blockNumber,
-        })
-      })
+    const marketAddress = metadata.marketAddress
+
+    // missing block number atm
```
Comment (bergarces) [inline packages/adapters-library/src/adapters/pendle/products/principle-token/pendlePrincipleTokenAdapter.ts:193]: ditto

**Review by bergarces — APPROVED** [review]

File: packages/adapters-library/src/adapters/pendle/products/lp-token/pendleLpTokenAdapter.ts (hunk 2 above)
Comment (jpsains) [reply packages/adapters-library/src/adapters/pendle/products/lp-token/pendleLpTokenAdapter.ts:192]: will fix comment in next PR.

**Pull request merged**